### DISCUSSION TITLE
Getting error while loading stitched ip of cyber security example in vivado 2022.2

### DISCUSSION THREAD

**Discussion by Rithikiitb** [Xilinx/finn#1043] [Q&A]
Hello, I have generated the stitched ip of the cyber security example through jupyter server but when i try to load this ip in the vivado i get critical warnings. what could be the issue?
![Screenshot from 2024-04-14 14-07-59](https://github.com/Xilinx/finn/assets/138851929/e0ac189c-909d-428e-aaaf-b2d8716afe66)

**Comment by fpjentzsch** [comment]
Hi,
the stitched-ip depends on the intermediate output products (sub IP cores) located in FINN_HOST_BUILD_DIR. Maybe these have been deleted or you are using the stitched-ip in a Vivado instance that does not have access to the build directory?

In any case, you could use the following builder option to generate a synthesized checkpoint (.dcp) for the stitched-ip, which makes it independent from the sub IP core definitions.
https://github.com/Xilinx/finn/blob/e188b4c50955105717b223862c4e26e4777852ea/src/finn/builder/build_dataflow_config.py#L221

**Reply by Rithikiitb** [reply]
Hello @fpjentzsch ,
That line is present and is false.
I have followed the tutorial of cybersecurity, i have not deleted any file and im running the vivado instance as root so it has access to all file. It is that file that the vivado is trying to search, are not created in that location. I have searched it too, those files are not created. But i could see the stitched ip (xpr file) which i am able to open but i am getting critical warnings saying files not found . can you please explain where i might have missed out.

**Reply by Rithikiitb** [reply]
hello @fpjentzsch 
I ran the docker again and generated the stitched ip, It opened without any critical warning but then when i opened it today. I got this critical warning 

[Project 1-19] Could not find the file '/tmp/finn_dev_project/code_gen_ipgen_StreamingFIFO_rtl_0_ux6mgu_w/StreamingFIFO_rtl_0.v'.

last time it got created in /home/project and now in /tmp and is not found. Why is it creating in tmp, it would however get deleted. please can you help in this.
